static/javascript/realtime.js

```diff
@@ -134,6 +135,28 @@ function checkForData() {
 }
 
 
+function initialHide(){
+    const http = new XMLHttpRequest();
+    const url = "/realtime/data";
+    http.open("GET", url);
+    http.send();
+
+    http.onreadystatechange = function () {
```

Comment:
As you can see in the below for loop over the `charts` array, the `data` received from the XmlHttpRequest is not used, so we can remove all the `http` code in this function and just keep the for loop.